charts/collabora-online/0.0.1/questions.yaml

```diff
@@ -0,0 +1,163 @@
+groups:
+  - name: "Container Image"
+    description: "Configure Container Image" 
+  - name: "Privileged"
+    description: "Privileged Configuration"
+  - name: "Networking"
+    description: "Network Configuration"
+  - name: "Environment Variables"
+    description: "Environment Variables Configuration"
+
+#Portal
+portals:
+  web_portal:
+    protocols:
+      - "https"
+    host: 
+      - "$node_ip"
+    ports:
+      - "$variable-service.port.nodePort"
+    path: "/loleaflet/dist/admin/admin.html"
+
+questions:
+  #Image related
+  - variable: image
+    group: "Container Image"
+    label: "Collabora"
+    schema:
+      type: dict
+      required: true
+      attrs:
+        #Image
+        - variable: repository
+          label: "Image Repository"
+          schema:
+            type: string 
+            default: "collabora/code"
+            editable: false
+        #Tag
+        - variable: tag
+          label: "Image Tag"
+          description: "Tag to use for specified image"
+          schema:
+            type: string
+            default: "6.4.6.2"
+            required: true
+        #Pull Policy
+        - variable: pullPolicy
+          label: "Image Pull Policy"
+          schema:
+            type: string
+            default: "IfNotPresent"
+            enum:
+              - value: "IfNotPresent"
+                description: "Only pull image if not present on host"
+              - value: "Always"
+                description: "Always pull image even if present on host"
+              - value: "Never"
+                description: "Never pull image even if it's not present on host"
+  # Strategy
+  - variable: strategyType
+    group: "Container Image"
+    label: "Update Strategy"
+    schema:
+      type: string
+      default: Recreate
+      enum:
+        - value: "RollingUpdate"
+          description: "Create new pods and then kill old ones"
+        - value: "Recreate"
+          description: "Kill existing pods before creating new ones"        
+
+  # Enable privileged
+  - variable: securityContext
+    group: "Privileged"
+    label: "Security Context"
+    schema:
+      type: dict
+      attrs:
+        - variable: privileged
+          label: "Enable privileged"
+          schema:
+            type: boolean
+            default: false
```

Comment:
We (generally) only offer privilaged mode for containers that require it by design.

Reply:
[https://www.collaboraoffice.com/code/docker/]()
Note: for the faster jail creation via bind mount, you need to use docker run command with the --privileged flag.

The above is mention in the collabora docker "wiki", but I'm not sure if privileged is needed here.

Reply:
You are using k8s, not docker.
Remember: All K8S settings (including storage) have nothing to do with your container, it has no say in how things are mounted.

That being said: there are some issues with storage that might require an initcontainer... but thats another story.

Reply:
I see! Removed privileged, Thanks for clarifing